paasta_tools/kubernetes_tools.py

```diff
@@ -3680,28 +3756,121 @@ def update_crds(
     return success
 
 
-def get_kubernetes_secret_name(
-    service_name: str, secret_name: str, namespace: str = "paasta"
+def sanitise_label_value(service_name: str) -> str:
+    """
+    :param service_name: service_name is sanitized and limited to 63 characters due to kubernetes restriction
+    :return: Sanitised at most 63-character label value
+    """
+    return limit_size_with_hash(
+        sanitise_kubernetes_name(service_name),
```

Comment:
final nit after our chat in slack:
```suggestion
def sanitise_label_value(value: str) -> str:
    """
    :param value: value is sanitized and limited to 63 characters due to kubernetes restriction
    :return: Sanitised at most 63-character label value
    """
    return limit_size_with_hash(
        sanitise_kubernetes_name(value),
```

Reply:
(not all label values are service names)